.zenodo.json

```diff
@@ -5,79 +5,128 @@
         {
             "affiliation": "Elemental Cognition", 
             "name": "Tom Breloff"
-        }, 
+        }
+    ],
+    "contributors":[
         {
             "affiliation": "TU Wien", 
-            "name": "Daniel Schwabeneder"
-            "orcid": "0000-0002-0412-0777"
+            "name": "Daniel Schwabeneder",
+            "orcid": "0000-0002-0412-0777",
+            "type": "ProjectLeader"
         }, 
         {
             "affiliation": "GLOBE Institute", 
-            "name": "Michael Krabbe Borregaard"
+            "name": "Michael Krabbe Borregaard",
+            "type": "ProjectLeader"
         }, 
         {
             "affiliation": "Leibniz Universit\u00e4t Hannover", 
             "name": "Simon Christ",
-            "orcid": "0000-0002-5866-1472"
+            "orcid": "0000-0002-5866-1472",
+            "type": "ProjectMember"
         }, 
         {
             "affiliation": "Forschungszentrum J\u00fclich", 
-            "name": "Josef Heinen"
+            "name": "Josef Heinen",
+            "orcid": "0000-0001-6509-1925",
+            "type": "ProjectLeader"
         }, 
         {
-            "name": "Yuval"
+            "name": "Yuval",
+            "type": "Other"
         }, 
         {
-            "name": "Andrew Palugniok"
+            "name": "Andrew Palugniok",
+            "type": "ProjectLeader"
         }, 
         {
             "affiliation": "@beacon-biosignals", 
-            "name": "Simon"
+            "name": "Simon",
+            "type": "Other"
         }, 
         {
             "affiliation": "Veos Digital (https://veos.digital/)", 
-            "name": "Pietro Vertechi"
+            "name": "Pietro Vertechi",
+            "type": "ProjectLeader"
         }, 
         {
-            "name": "Zhanibek"
+            "name": "Zhanibek",
+            "type": "Other"
```

Comment:
name: Zhanibek Omarov
orcid: https://orcid.org/0000-0002-8783-8791

aff: Korea Advanced Inst. of Science and Technology (KAIST)

Reply:
I'd classify you as "ProjectMember" if you don't have objections

Reply:
That is good